packages/tables/src/elements/Row.tsx

```diff
@@ -54,10 +57,14 @@ export const Row = forwardRef<HTMLTableRowElement, IRowProps>(
       <StyledRow
         onFocus={onFocusCallback}
         onBlur={onBlurCallback}
-        size={size}
-        isReadOnly={isReadOnly}
-        isFocused={computedFocused}
+        $size={size}
+        $isReadOnly={isReadOnly}
+        $isFocused={computedFocused}
+        $isHovered={isHovered}
+        $isStriped={isStriped}
+        $isSelected={isSelected}
         ref={ref}
+        tabIndex={isReadOnly ? undefined : -1}
```

Comment:
I like the element component move, but in order to match previous the `tabIndex` should follow `{...otherProps}` so that it wins.